crates/smdk/src/build.rs

```diff
@@ -0,0 +1,45 @@
+use anyhow::{Error, Result};
+use clap::Parser;
+use std::process::Command;
+use std::fmt::Debug;
+
+/// Builds the SmartModule in the current working directory into a WASM file
+#[derive(Debug, Parser)]
+pub struct BuildOpt;
+
+impl BuildOpt {
+    pub(crate) fn process(&self) -> Result<()> {
+        BuildOpt::check_cargo_exists()?;
+
+        let mut cmd = Command::new("cargo");
+        let cwd = std::env::current_dir()?;
+
+        cmd.current_dir(&cwd)
+            .arg("build")
+            .arg("--profile")
```

Comment:
make profile as optional so it can be overridden.  see TestOption.  Default should be made as crate constant